src/main/template/vpc.yaml

```diff
@@ -0,0 +1,442 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: This template deploys a VPC. Its IP range is quartered into four parts. The first three parts form private
+  subnets, the fourth is again quartered into three public subnets. Both the private and the public subnets are then
+  spread across three Availability Zones. All the required infrastructure, like internet gateway, nat gateways, and
+  default routes are included. Also, a bastion jump host is deployed in order to make any private machines reachable.
+
+Parameters:
+  cidrBlock:
+    Description: The CIDR range of the VPC. Note that it cannot be larger than 16 bits.
+    Type: String
+    Default: 10.0.0.0/16
+
+  firstQuarteringPartitionBits:
+    Description: The bit size of the first quartering, which is the size of the three private subnets. This is the size
+      of the CIDR block minus two. Unfortunately, CloudFormation does not support arithmetics, so this needs to be
+      passed in as a parameter.
+    Type: String
+    Default: 14
+
+  secondQuarteringPartitionBits:
+    Description: The bit size of the second quartering, which is the size of the three public subnets. This is the size
```

Comment:
With [schroenser's explanation above](https://github.com/mizool/cloudformation/pull/1/files#r347250720) in mind, I suggest replacing the description with this:

> The cidr bits for the four subnets on the second level ("public subnet A", "public subnet B", "public subnet C", "unused").